source/lifecycle/mattermost-mobile-releases.md

```diff
@@ -0,0 +1,146 @@
+# Mattermost mobile app releases
+
+```{include} ../_static/badges/allplans-cloud-selfhosted.md
+```
+
+```{Important}
+**Support for Mattermost Server v8.1 Extended Support Release has come to the end of its life cycle on May 15, 2024. Upgrading to Mattermost Server v9.5 Extended Support Release or later is required.**
+```
+
+## Frequency
+
+Mattermost releases a new mobile app version every 4 months, in February, May, August, and November in binary form. Mattermost tests the latest mobile app release against the [latest server version](/lifecycle/mattermost-server-releases).
+
+See the [Mobile app changelog](/lifecycle/mobile-app-changelog) for release details, and see the [iOS mobile app](/collaborate/install-ios-app) and the [Android mobile app](/collaborate/install-android-app) documentation for installation details. 
+
+## Latest releases
+
+| **Release** | **Life cycle start** | **Life cycle end** |
```

Comment:
@amyblais - Can I get your help to populate the correct life cycle start and end dates for these mobile releases (based on our mobile changelog content) please?

Reply:
Done for v2.x version - propose we remove v1.x versions as they are no longer supported.

Reply:
Great idea, @amyblais! Would you be open to removing the v1.x version content?